libraries/redcore/api/hal/model/item.php

```diff
@@ -123,7 +125,15 @@ public function getItem($pk = null)
 
 		if (property_exists($item, 'params'))
 		{
-			$registry = new Registry;
+			if (version_compare(JVERSION, '3.3', '<'))
```

Comment:
maybe just use plain old JRegistry instead since it has an alias in 3.4? or make an alias in redCORE for Registry, otherwise we will have those switches all around. I am up for some more permanent and easier solution than version check :)